src/Fractions/Fraction.cs

```diff
@@ -146,4 +153,180 @@ namespace Fractions;
+    /// </returns>
+    static bool INumberBase<Fraction>.IsInfinity(Fraction value) {
+        return value.IsInfinity;
+    }
+
+    /// <summary>Determines if a value is positive infinity.</summary>
+    /// <param name="value">The value to be checked.</param>
+    /// <returns>
+    ///     <see langword="true" /> if <paramref name="value" /> is positive infinity; otherwise, <see langword="false" />.
+    /// </returns>
+    static bool INumberBase<Fraction>.IsPositiveInfinity(Fraction value) {
+        return value.IsPositiveInfinity;
+    }
+
+    /// <summary>Determines if a value is negative infinity.</summary>
+    /// <param name="value">The value to be checked.</param>
+    /// <returns>
+    ///     <see langword="true" /> if <paramref name="value" /> is negative infinity; otherwise, <see langword="false" />.
+    /// </returns>
+    static bool INumberBase<Fraction>.IsNegativeInfinity(Fraction value) {
+        return value.IsNegativeInfinity;
+    }
+
+    /// <summary>Determines if a value is NaN.</summary>
+    /// <param name="value">The value to be checked.</param>
+    /// <returns>
+    ///     <see langword="true" /> if <paramref name="value" /> is NaN; otherwise, <see langword="false" />.
+    /// </returns>
+    static bool INumberBase<Fraction>.IsNaN(Fraction value) {
+        return value.IsNaN;
+    }
+#endif
+
+    /// <summary>
+    ///     Determines whether the given Fraction is in canonical form.
+    /// </summary>
+    /// <param name="value">The Fraction to check.</param>
+    /// <returns>True if the Fraction is in canonical form, otherwise false.</returns>
+    /// <remarks>
+    ///     A Fraction is considered to be in canonical form if its denominator is one,
+    ///     or if its numerator and denominator are coprime numbers (their greatest common divisor is one).
+    /// </remarks>
+    public static bool IsCanonical(Fraction value) {
+        var numerator = value.Numerator;
+        var denominator = value.Denominator;
+        if (denominator.IsOne) {
+            return true;
+        }
+
+        if (numerator.IsZero) {
+            return denominator.IsZero; // if we want to consider NaN as "canonical"
+        }
+
+        if (denominator.IsZero) {
+            return numerator.IsOne || numerator == BigInteger.MinusOne;
+        }
+
+        return BigInteger.GreatestCommonDivisor(numerator, denominator).IsOne;
+    }
+
+    /// <summary>Determines if a value represents an even integral number.</summary>
+    /// <param name="value">The value to be checked.</param>
+    /// <returns>
+    ///     <see langword="true" /> if <paramref name="value" /> is an even integer; otherwise, <see langword="false" />.
+    /// </returns>
+    public static bool IsEvenInteger(Fraction value) {
+        var fraction = value.Reduce();
+        return fraction.Denominator.IsOne && fraction.Numerator.IsEven;

```

Comment:
I couldn't find any unit tests for this. What do you think about the following optimization?

```suggestion
        if (value.Denominator.IsZero) {
            return false;
        }

        if (!value._normalizationNotApplied) {
            return BigInteger.Abs(value.Denominator).IsOne && value.Numerator.IsEven;
        }

        var intValue = BigInteger.DivRem(value.Numerator, value.Denominator, out var remainder);
        return remainder.IsZero && intValue.IsEven;
```

Reply:
I'm looking at my other project and that's exactly how I've implemented it there. I've even left myself a little comment:

``` csharp
        // TODO see about first checking the magnitudes
        var quotient = BigInteger.DivRem(numerator, denominator, out var remainder);
        return remainder.IsZero && quotient.IsEven;
```

Reply:
fixed in https://github.com/Fractions-DotNet/Fractions/pull/96/commits/1643913092e03ca145a91e4d5de95948f16949da